data/presets/emergency/fire_service_inlet.json

```
@@ -0,0 +1,22 @@
+{
+    "icon": "temaki-fire_hydrant",
+    "fields": [
+        "fire_mains"
+    ],
+    "moreFields": [
+        "level",
+        "survey/date"
+    ],
+    "geometry": [
+        "point",
+        "vertex"
+    ],
+    "terms": [
+        "Building Hydrant Inlet",
+        "Fire Service Inlet"
```

Terms should be in lower case. I'd propose to add a few more entries:

```suggestion
        "building hydrant inlet",
        "charged riser",
        "dry riser inlet",
        "dry standpipe",
        "fdc",
        "fire service inlet",
        "fire sprinkler inlet",
        "riser inlet",
        "wet riser inlet"
```